flags.go

```diff
@@ -187,6 +187,12 @@ func newApp() (app *cli.App) {
 				Usage: "File chunk size to read from GCS in one call. Need to specify the value in MB. ChunkSize less than 1MB is not supported",
 			},
 
+			cli.BoolFlag{
+				Name: "disable-auth",
```

Comment:
=>
Authentication is enabled by default. The --disable-auth flag disables authentication. For users of the --custom-endpoint flag, please pass --disable-auth flag explicitly if you do not want authentication enabled for your workflow.